src/devices/Bmxx80/Bmx280Base.cs

```diff
@@ -200,34 +213,12 @@ public void SetPowerMode(Bmx280PowerMode powerMode)
             _i2cDevice.Write(new[] { _controlRegister, (byte)(cleared | (byte)powerMode) });
         }
 
-        /// <summary>
-        /// Sets the standby time mode the device will used when operating in normal mode.
-        /// </summary>
-        /// <param name="standbyTime">The <see cref="StandbyTime"/> to set.</param>
-        public void SetStandbyTime(StandbyTime standbyTime)
-        {
-            byte current = Read8BitsFromRegister((byte)Bmx280Register.CONFIG);
-            current = (byte)((current & 0b_0001_1111) | (byte)standbyTime << 5);
-            _i2cDevice.Write(new[] { (byte)Bmx280Register.CONFIG, current });
-        }
-
-        /// <summary>
-        /// Reads the currently configured standby time mode the device will used when operating in normal mode.
-        /// </summary>
-        /// <returns>The current <see cref="StandbyTime"/>.</returns>
-        public StandbyTime ReadStandbyTime()
-        {
-            byte current = Read8BitsFromRegister((byte)Bmx280Register.CONFIG);
-
-            return (StandbyTime)((current & 0b_1110_0000) >> 5);
-        }
-
         /// <summary>
         /// Recommended wait timings from the datasheet.
         /// </summary>
         /// <param name="sampleMode">The <see cref="Sampling"/> to get for.</param>
         /// <returns>The time it takes for the chip to read data in milliseconds rounded up.</returns>
-        internal int GetMeasurementTimeForForcedMode(Sampling sampleMode)
+        internal static int GetMeasurementTimeForForcedMode(Sampling sampleMode)
```

Comment:
why did we have to change this to static? all places where we call it we are already in an instance method.

Reply:
You're right this is actually not necessary.